test/e2e/Tests/Tests/TerminateOrchestratorTests.cs

```diff
@@ -0,0 +1,159 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Net;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.Azure.Durable.Tests.DotnetIsolatedE2E;
+
+[Collection(Constants.FunctionAppCollectionName)]
+public class TerminateOrchestratorTests
+{
+    private readonly FunctionAppFixture _fixture;
+    private readonly ITestOutputHelper _output;
+
+    public TerminateOrchestratorTests(FunctionAppFixture fixture, ITestOutputHelper testOutputHelper)
+    {
+        _fixture = fixture;
+        _fixture.TestLogs.UseTestLogger(testOutputHelper);
+        _output = testOutputHelper;
+    }
+
+    private static async Task AssertTerminateRequestFails(HttpResponseMessage terminateResponse)
```

Comment:
nit: we typically try to declare private methods below the public methods. This is what static analysis tools for C# like StyleCop enforce. I recommend we do the same here so that the most important code (the test code) is the first set of things we see.

Reply:
Done